deployments/k8s/helm/signalfx-agent/templates/configmap.yaml

```diff
@@ -117,6 +117,8 @@ data:
       timeout: {{ .Values.timeout | default 40 }}
       logLevel: {{ .Values.logLevel | default "notice" }}
 
+    enableBuiltInFiltering: true
```

Comment:
I wonder if this should be put into values.yaml and conditionalized to false for those upgrading from a previous helm release? ie only turn it on by deafult if it's a non-upgrade and for existing installs they can override it to true when they're ready to transition. This could be checked with `Release.IsUpgrade`

Reply:
Fortunately in the Helm configmap we never let people pass through `metricsToInclude` anywhere (they would have to provide their own config in its entirety instead of relying on templating in this file).  Wasn't the whole reason we went with the flag because of uncertainties around `metricsToInclude` backwards compatibility?  Or was there something else I am forgetting?

Reply:
Yep that should be safe then.